rcl_yaml_param_parser/QUALITY_DECLARATION.md

```diff
@@ -118,13 +135,13 @@ It also has several test dependencies, which do not affect the resulting quality
 
 `rcutils` provides commonly used functionality in C.
 
-It is **Quality Level 4**, see its [Quality Declaration document](https://github.com/ros2/rcutils/blob/master/QUALITY_DECLARATION.md).
+It is **Quality Level 3**, see its [Quality Declaration document](https://github.com/ros2/rcutils/blob/foxy/QUALITY_DECLARATION.md).
 
 #### `libyaml_vendor`
 
 `libyaml_vendor` is a vendor package for the libyaml C library.
 
-It is **Quality Level 4**, see its [Quality Declaration document](https://github.com/ros2/libyaml_vendor/blob/master/QUALITY_DECLARATION.md).
+It is **Quality Level 4**, see its [Quality Declaration document](https://github.com/ros2/libyaml_vendor/blob/foxy/QUALITY_DECLARATION.md).
```

Comment:
Maybe we should wait for https://github.com/ros2/libyaml_vendor/pull/20 to get in and be backported, so we can bump `rcl_yaml_param_parser` straight to level 1.

Reply:
That PR is in and backported. So `rcl_yaml_param_parser` can be bumped to level 1 on Foxy and Rolling :top:

Reply:
Actually it can only go up to level 2, because of `rcutils`. See my other comments.